frontend/src/views/CellCards/components/CellCard/components/OntologyDagView/components/AnimatedLinks/index.tsx

```diff
@@ -0,0 +1,111 @@
+import NodeGroup from "react-move/NodeGroup";
+import { HierarchyPointNode } from "@visx/hierarchy/lib/types";
+import { LinkHorizontal } from "@visx/shape";
+import { Group } from "@visx/group";
+import { TreeNodeWithState } from "../../common/types";
+import { secondaryColor } from "../../common/constants";
+
+interface AnimatedLinksProps {
+  tree: HierarchyPointNode<TreeNodeWithState>;
+  duration: number;
+}
+
+export default function AnimatedLinks({ tree, duration }: AnimatedLinksProps) {
+  return (
+    <NodeGroup
+      data={tree.links()}
+      keyAccessor={(d) => `${d.source.data.id}_${d.target.data.id}`}
+      start={({ source }) => {
+        return {
+          source: {
+            x: source.data.x0 ?? source.x,
```

Comment:
"x0" is the x position of the link at the start of the animation. "x" is the current x position of the link.